charts/puris/values.yaml

```diff
@@ -339,36 +352,37 @@ backend:
             password:
         edc:
             controlplane:
-                # -- IP address of the EDC control plane
-                host: 172.17.0.2
-                data:
-                    # -- Data port of the EDC control plane
-                    port: 30091
-                # -- Key for the EDC control plane
-                key: ""
-            backend:
-                # -- URL of the EDC backend service
-                url: http://172.17.0.2:32084
-            applydataplaneworkaround: true
-            dataplane:
-                public:
-                    # -- Public port of the EDC data plane
-                    port: 30784
-            # -- Own IDS URL of the EDC
-            idsUrl: http://plato-controlplane:8084/api/v1/ids
-        minikube:
-            # -- IP of minikube for the EDC configuration
-            ip: host.minikube.internal
+              host: "172.17.0.2"
+              management:
+                # -- Url to the EDC controlplane management of the edc
+                url: "https:/your-edc-address:your-port/management"
+              protocol:
+                # -- Url to the EDC controlplane protocol API of the edc
+                url: "https://your-edc-address:your-port/api/v1/dsp"
```

Comment:
I've had a constraint violation in my backend. Reason is that we evaluate that it's a valid uri. As ":your-port" does not match the format, the application will throw several errors. Thus, I propose a small further change. For the two addresses use management port 8181 and supplier port 8184.